server/sandbox_run_linux.go

```diff
@@ -444,6 +444,14 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		g.AddLinuxSysctl(key, value)
 	}
 
+	devices, err := getDevicesFromConfig(s.config)
+	if err != nil {
+		return nil, err
+	}
+	for _, d := range devices {
+		g.AddDevice(d)
```

Comment:
let's do this just for containers, sandboxes processes should not have this, just application containers